File: Aulas PA_JavaScript/java & outros/ordenar.cpp
#include <iostream>

int main(){
    int a, b, c;
    std::cout << "Digite o número 1: ";
    std::cin >> a;
    std::cout << "Digite o número 2: ";
    std::cin >> b;
    std::cout << "Digite o número 3: ";
    std::cin >> c;

    if (a < b && a < c) {
        if (b < c) {
            std::cout << a << " " << b << " " << c << std::endl;
        } else {
            std::cout << a << " " << c << " " << b << std::endl;
        }
    } else if (b < a && b < c) {
        if (a < c) {
            std::cout << b << " " << a << " " << c << std::endl;
        } else {
            std::cout << b << " " << c << " " << a << std::endl;
        }
    } else if (c < a && c < b) {
        if (a < b) {
            std::cout << c << " " << a << " " << b << std::endl;
        } else {
            std::cout << c << " " << b << " " << a << std::endl;
        }
    }
}

File: Aulas PA_JavaScript/java & outros/Ordenar.java
import java.util.*;

public class Ordenar {
    public static void main(String[] args){
        Scanner sc = new Scanner (System.in);

        System.out.print("digite o número 1: ");
        double a = sc.nextDouble();

        System.out.print("digite o número 2: ");
        double b = sc.nextDouble();

        System.out.print("digite o número 3: ");
        double c = sc.nextDouble();


                        if (a <= b && a <= c) {
            if (b <= c) {
                System.out.println(a + " " + b + " " + c);
            } else {
                System.out.println(a + " " + c + " " + b);
            }
        } else if (b <= a && b <= c) {
            if (a <= c) {
                System.out.println(b + " " + a + " " + c);
            } else {
                System.out.println(b + " " + c + " " + a);
            }
        } else {
            if (a <= b) {
                System.out.println(c + " " + a + " " + b);
            } else {
                System.out.println(c + " " + b + " " + a);
            }
        }
            sc.close();
        
    }
}